doc/releases/release-notes-2.7.rst

```diff
@@ -81,6 +81,10 @@ Modified in this release
 
 * The ``CONFIG_BT_PERIPHERAL_PREF_SLAVE_LATENCY`` Kconfig option is now
   :kconfig:`CONFIG_BT_PERIPHERAL_PREF_LATENCY`
+* The ``CONFIG_BT_CTLR_SLAVE_FEAT_REQ_SUPPORT`` Kconfig option is now
+  :kconfig:`CONFIG_BT_CTLR_PER_INIT_FEAT_XCHG_SUPPORT`
```

Comment:
@wopu-ot we had this conversation when we changed the hci API:
https://github.com/zephyrproject-rtos/zephyr/pull/38605#pullrequestreview-756486084

I think that this is OK in this case, since it is very easy to map to the spec.

Reply:
`PER` is used for naming Peripheral Test Cases in the Bluetooth Core Test Specification document.